packages/site/src/components/RpcConnect.tsx

```diff
@@ -0,0 +1,35 @@
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import React, { useState } from "react";
+import Typography from "@mui/material/Typography";
+
+type Props = {
+  url: string;
+  setUrl: (url: string) => void;
+};
+
+export default function RpcConnect({ url, setUrl }: Props) {
+  const [urlToEdit, setUrlToEdit] = useState(url);
+  function handleChange(e: React.ChangeEvent<HTMLInputElement>) {
+    e.preventDefault();
```

Comment:
What does this line do? I think in the other handler, it prevents the page from reloading. But not sure about here?